tests/lora/utils.py

```diff
@@ -126,13 +126,26 @@ class PeftLoraLoaderMixinTests:
     text_encoder_target_modules = ["q_proj", "k_proj", "v_proj", "out_proj"]
     denoiser_target_modules = ["to_q", "to_k", "to_v", "to_out.0"]
 
-    def get_dummy_components(self, use_dora=False, lora_alpha=None):
+    cached_non_lora_output = None
+
+    @pytest.fixture(scope="class", autouse=True)
```

Comment:
Think this might be a leftover from previous refactoring. It's not used anywhere. Think it can be removed.